TINY-7399: Context menu not closing when clicking into a different editor (#7015)
tinymce/tinymce: modules/tinymce/CHANGELOG.md

- Links in notification text did not show the correct mouse pointer #TINY-7661
- The formatter match APIs were incorrectly returning false for formats that specified an attribute or style should be removed from an element #TINY-6149
- The type signature on `editor.formatter.matchNode` had the wrong return type (was `boolean` but should have been `Formatter | undefined`) #TINY-6149

> - Context menu not closing when clicking into a different editor #TINY-7399
> - Menu not closing when clicking into a different editor or opening other editor menu #TINY-7399

Reviewer: ```suggestion
- Menus and context menus were not closed when clicking into a different editor #TINY-7399
```
This only needs a single changelog entry.
